### ISSUE TITLE
Vault Reader methods "getImplicit() getPrefix() getSuffix()" throw exceptions

### ISSUE THREAD

**Issue by JarekZagorski** [Odgug/CC-Vault#1]
Calling any of these methods:
```lua
getImplicit() 
getPrefix() 
getSuffix()
```

results in following exception:
```
[03:36:39] [ComputerCraft-Coroutine-4/ERROR] [computercraft/]: Error calling call on dan200.computercraft.core.apis.PeripheralAPI@3b29481a
java.lang.NoSuchMethodError: 'java.util.Optional iskallia.vault.gear.attribute.VaultGearModifier.getDisplay(iskallia.vault.gear.data.VaultGearData, iskallia.vault.gear.attribute.VaultGearModifier$AffixType, net.minecraft.world.item.ItemStack, boolean)'
	at net.joseph.ccvault.peripheral.custom.VaultReaderBlockPeripheral.getPrefix(VaultReaderBlockPeripheral.java:214) ~[ccvault-1.0.0-3.12.4.jar%2363!/:1.0.0-3.12.4] {re:classloading}
	at net.joseph.ccvault.peripheral.custom.VaultReaderBlockPeripheral$cc$getPrefix17.apply(CC generated method) ~[?:?] {re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading}
	at dan200.computercraft.core.apis.PeripheralAPI$PeripheralWrapper.call(PeripheralAPI.java:112) ~[cc-tweaked-1.18.2-1.101.3.jar%2362!/:1.101.3] {re:classloading}
	at dan200.computercraft.core.apis.PeripheralAPI.call(PeripheralAPI.java:361) ~[cc-tweaked-1.18.2-1.101.3.jar%2362!/:1.101.3] {re:classloading}
	at dan200.computercraft.core.apis.PeripheralAPI$cc$call71.apply(CC generated method) ~[?:?] {re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading,re:classloading}
	at dan200.computercraft.core.lua.ResultInterpreterFunction.invoke(ResultInterpreterFunction.java:61) ~[cc-tweaked-1.18.2-1.101.3.jar%2362!/:1.101.3] {re:classloading}
	at cc.tweaked.internal.cobalt.function.ResumableVarArgFunction.invoke(ResumableVarArgFunction.java:77) ~[cc-tweaked-1.18.2-1.101.3.jar%2362!/:1.101.3] {re:classloading}
	at cc.tweaked.internal.cobalt.function.LuaInterpreter.execute(LuaInterpreter.java:506) ~[cc-tweaked-1.18.2-1.101.3.jar%2362!/:1.101.3] {re:classloading}
	at cc.tweaked.internal.cobalt.function.LuaInterpretedFunction.invoke(LuaInterpretedFunction.java:149) ~[cc-tweaked-1.18.2-1.101.3.jar%2362!/:1.101.3] {re:classloading}
	at cc.tweaked.internal.cobalt.LuaThread.loop(LuaThread.java:523) ~[cc-tweaked-1.18.2-1.101.3.jar%2362!/:1.101.3] {re:classloading}
	at cc.tweaked.internal.cobalt.LuaThread$1.run(LuaThread.java:466) ~[cc-tweaked-1.18.2-1.101.3.jar%2362!/:1.101.3] {re:classloading}
	at dan200.computercraft.core.lua.CobaltLuaMachine.lambda$new$1(CobaltLuaMachine.java:80) ~[cc-tweaked-1.18.2-1.101.3.jar%2362!/:1.101.3] {re:classloading}
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?] {}
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?] {}
	at java.lang.Thread.run(Thread.java:833) [?:?] {re:mixin}
```

Example code required to replicate the issue:
```lua
local vaultReader = peripheral.find("vaultreader")

vaultReader.getPrefix(1)
```
Tested on several gear pieces

**Comment by Odgug** [comment]
Oh i was dm'ed the issue aswell, will release a fix soon

**Comment by Odgug** [comment]
Released version 1.0.1 with the fix, ty for reporting this and sorry for that issue
